internal/runners/projects/projects.go

```diff
@@ -16,7 +17,31 @@ import (
 type projectWithOrg struct {
 	Name           string   `json:"name"`
 	Organization   string   `json:"organization"`
-	LocalCheckouts []string `json:"local_checkouts,omitempty" locale:"local_checkouts,Local Checkouts" opts:"emptyNil,singleLine"`
+	LocalCheckouts []string `json:"local_checkouts,omitempty"`
+}
+
+type projectWithOrgs []projectWithOrg
+
+func (o projectWithOrgs) MarshalOutput(f output.Format) interface{} {
+	if f != output.PlainFormatName {
+		return o
+	}
+
+	r := []projectOutputPlain{}
+	for _, v := range o {
+		checkouts := []string{}
+		for _, checkout := range v.LocalCheckouts {
+			checkouts = append(checkouts, locale.Tl("projects_local_chekcout", " └─ ✔ Local Checkout → {{.V0}}", checkout))
+		}
+		r = append(r, projectOutputPlain{v.Name, v.Organization, strings.Join(checkouts, "\n")})
+	}
+	return r
+}
+
+type projectOutputPlain struct {
+	Name           string
+	Organization   string
+	LocalCheckouts string `locale:"local_checkouts,Local Checkouts" opts:"emptyNil,separateLine"`
 }
```

Comment:
Can you move this type definition into the `MarshalOutput` function? It doesn't appear to be used anywhere else